assembly/activities/createNodesFromSamples.js

```diff
@@ -44,7 +44,12 @@ export async function createNodesFromSamples(config) {
       writeStr = `${writeStr}title: ${node.metadata.title}\n`;
       writeStr = `${writeStr}sidebar_label: ${node.metadata.label}\n`;
       writeStr = `${writeStr}description: ${node.metadata.description}\n`;
+      writeStr = `${writeStr}tags:\n`;
+      for (const tag of node.metadata.tags) {
+        writeStr = `${writeStr}- ${tag}\n`;
+      }
       writeStr = `${writeStr}---\n\n`;
+      writeStr = `${writeStr}<!-- THIS FILE IS GENERATED from ${sourceURL}.\nDO NOT EDIT THIS FILE DIRECTLY -->\n\n`;
```

Comment:
What do you think about leading with the directive? That is, "DO NOT EDIT..." followed by "This file is generated from...."